common/src/main/java/com/wynntils/core/functions/FunctionManager.java

```diff
@@ -317,6 +317,11 @@ public static void init() {
         registerFunction(new CharacterFunctions.XpPctFunction());
         registerFunction(new CharacterFunctions.BpsFunction());
         registerFunction(new CharacterFunctions.BpsXzFunction());
+        registerFunction(new CharacterFunctions.LiquidEmeraldFunction());
```

Comment:
Put these in alphabetical order. Make sure to move bps too, they break the order atm

Reply:
How is this resolved? It still looks very much out of order.